### PR TITLE
New run queue service in example (2)

### PR DESCRIPTION
**Pull request by mtwharmby**
Another attempt to include the bridging class. This also attempts to fix the dodgy RemoteQueueControllerServiceTest.

Signed-off-by: Michael Wharmby <michael.wharmby@diamond.ac.uk>

### CODE REVIEW

**Comment by Mark-Booth** [comment]
The travis build 20 minutes ago failed, so I've just restarted it to see if the error is transient or not.

**Comment by mtwharmby** [comment]
It passed, so looks transient. I've restarted again to see if I really have managed to squash the problem with RemoteQueueControllerServiceTest.

**Comment by mtwharmby** [comment]
It past again after failing. I will restart it again to see if it's "fixed" *again*

**Comment by mtwharmby** [comment]
I've built this four times now without it failing. I think this *should* resolve the RemoteQueueControllerServiceTest flakiness. @gerring

**Comment by Mark-Booth** [comment]
@gerring , are you happy for this PR to be merged?

File: org.eclipse.scanning.event/src/org/eclipse/scanning/event/remote/_QueueControllerService.java [hunk 1]
```diff
@@ -176,15 +176,15 @@ public void stopQueueService(boolean force) throws EventException {
...
 			throw evEx;
 		}
 		if (beanState.isResumed() || beanState.isRunning()) {
-			logger.error("Bean '"+bean.getName()+"' is already resumed/running.");
+			logger.error(bean.getClass().getSimpleName()+" '"+bean.getName()+"' is already resumed/running.");
 			throw new IllegalStateException("Bean '"+bean.getName()+"' is already resumed/running");
 		} else if (beanState == Status.SUBMITTED) {
-			logger.error("Bean is submitted but not being processed. Cannot resume.");
-			throw new IllegalStateException("Cannot resume a bean with SUBMITTED status");
+			logger.error(bean.getClass().getSimpleName()+" '"+bean.getName()+"' is submitted but not being processed. Cannot resume.");
```
Comment (gerring) [inline org.eclipse.scanning.event/src/org/eclipse/scanning/event/remote/_QueueControllerService.java]: The state testing in QueueControllerService.java and _QueueControllerService looks like repeated code. Is it possible to somehow do this in one place.

**Comment by gerring** [comment]
The code for the exception handling looks like repeated stuff to me. I prefer that it is done once. Otherwise the new test seems to add helpful checking. Is it definitely included in a Suite?

**Comment by mtwharmby** [comment]
New push moves all the logging stuff into default methods on IQueueControllerService interface (and an inner interface of that). This was the only way to move this code into a common place since the inheritance hierarchy of the local and remove implementations of the service are different.

**Comment by gerring** [comment]
Alright to merge this now?

**Comment by mtwharmby** [comment]
Cheers @gerring !

**Pull request merged**